test/e2e/utils.go

```diff
@@ -172,20 +173,21 @@ func run(node string, command ...string) {
 	ssh_command := []string{"ssh", node}
 	ssh_command = append(ssh_command, command...)
 	cmd := exec.Command("kubevirtci/cluster-up/cli.sh", ssh_command...)
+	GinkgoWriter.Write([]byte(strings.Join(ssh_command, " ") + "\n"))
 	cmd.Stdout = GinkgoWriter
 	cmd.Stderr = GinkgoWriter
 	Expect(cmd.Run()).To(Succeed())
 }
 
 func createDummy(nodes []string, dummyName string) {
 	for _, node := range nodes {
-		run(node, "sudo", "ip", "link", "add", dummyName, "type", "dummy")
+		run(node, "sudo", "nmcli", "con", "add", "type", "dummy", "con-name", dummyName, "ifname", dummyName)
```

Comment:
how is this related to the PR?

Reply:
stdci was failing at "ip link add" with "RTNETLINK answers: File exists" but the interface was not there, looks like it's relatd to flushing the interface, doing it with nmcli works and also kind of make sense since we are testing networkmanager.

So this is all to make CI happy.